troi/filters.py

```diff
@@ -110,3 +111,32 @@ def read(self, inputs, debug=False):
                 results.append(r)
 
         return results
+
+
+class DuplicateRecordingFilterElement(troi.Element):
```

Comment:
We added a filter to AB so that it would remove items that had the same mbid and same distance score (but a different offset).
What I found is that we have some duplicate submissions with a very similar (but not exact) score. 
This filter was an easy way to remove these dups, and could be useful in other pipelines. We should also add another option to AB to filter these out as well